apps/condo/domains/onboarding/hooks/TourContext/useCompletedTourModals.tsx

```diff
@@ -0,0 +1,407 @@
+import { TourStepStatusType, TourStepTypeType } from '@app/condo/schema'
+import styled from '@emotion/styled'
+import { Col, Row } from 'antd'
+import get from 'lodash/get'
+import isEmpty from 'lodash/isEmpty'
+import pickBy from 'lodash/pickBy'
+import Link from 'next/link'
+import { useRouter } from 'next/router'
+import React, { useCallback, useMemo, useState } from 'react'
+
+import { Building, Guide, LayoutList, Meters, Unlock, Wallet } from '@open-condo/icons'
+import { useIntl } from '@open-condo/next/intl'
+import { useOrganization } from '@open-condo/next/organization'
+import { Button, Modal, Space, Typography } from '@open-condo/ui'
+import { colors } from '@open-condo/ui/dist/colors'
+
+import { FocusContainer } from '@condo/domains/common/components/FocusContainer'
+import { useTracking } from '@condo/domains/common/components/TrackingContext'
+import { TourStep } from '@condo/domains/onboarding/utils/clientSchema'
+import { EXTERNAL_GUIDE_LINK } from '@condo/domains/onboarding/utils/clientSchema/constants'
+
+
+type ButtonClickType = () => void
+
+type CompletedStepModalDataValueType = {
+    type?: TourStepTypeType | 'importProperties'
```

Comment:
No, `importProperties` has custom type because it is a `createProperty` and `createPropertyMap` step types.
Other imports have same completed step modals data as create action from form